elements/eos/eos-license-service.bst

```diff
@@ -0,0 +1,12 @@
+kind: autotools
+description: |
+   GJS service that extracts all the license files on demand.
+
+sources:
+- kind: git_repo
+  url: github:endlessm/eos-license-service.git
+  track: Release_*
+  ref: Release_6.0.7-0-g56aba79154be06970fd3795f3a801d6d0676c208
+
+build-depends:
+  - freedesktop-sdk.bst:public-stacks/buildsystem-autotools.bst
```

Comment:
eos-license-service uses featurer provided by gjs and systemd.  So, please add gjs.bst and systemd.bst into runtime-depends.